src/hooks/useSnackbar/stories-helpers.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useRef } from 'react';
+
+import { useSnackbar } from './useSnackbar';
+import { Button } from '../../components/basic/Button';
+import { CreateSnackbarFnArgs } from '../../components/utilities/SnackbarManager';
+
+export const Hook = (createSnackbarArgs: CreateSnackbarFnArgs): React.JSX.Element => {
```

Comment:
Isn't it better to give it a better and explicative name? This seems to me too generic but I could miss some logic behind it

Reply:
I could name it UseSnackbarStoryComponent. The purpose is just to make storybook read the props in order to document the hook args. I had to export it from a file different from the .stories.tsx because otherwise it would have been recognized as a story.

Reply:
If the current name has been already accepted by others I think it is fine to keep it as it is.
My preference would be to go with ur suggested name UseSnackbarStoryComponent and UseSnackbarStoryComponentExample for the usage case but the already selected ones are fine too